src/components/AppShell.tsx

```
@@ -10,6 +11,49 @@ if (import.meta.env.VITE_DEMO_MODE === 'true' && typeof window !== 'undefined')
+        {' '}or see the{' '}
+        <a
+          href="https://github.com/jaredglaser/homelab-manager/blob/main/self-hosting/README.md"
+          target="_blank"
+          rel="noopener noreferrer"
+          className="underline font-medium"
+        >
+          self-hosting guide
+        </a>
+        {' '}to run it against your own infrastructure.
+      </span>
+      <button onClick={dismiss} aria-label="Dismiss demo banner" className="shrink-0 opacity-80 hover:opacity-100 cursor-pointer text-lg leading-none">
```

Biome (2.4.4)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)